lib/registry.js

```diff
@@ -254,8 +266,10 @@ class RegistryFetcher extends Fetcher {
                 ), { code: 'EMISSINGSIGNATUREKEY' })
               }
 
+              const publishTime = mani._time ?
```

Comment:
We could pull a more reliably time from the bundle's integrated time when verifying attestations, which I believe is signed

cc @bdehamer thoughts on this?

Reply:
Yeah, assuming that the we want the timestamp that is as close as possible to the exact moment when the signature was generated, the `bundle.verificationMaterial.tlogEntries[0].integratedTime` is gonna be more accurate. I looked at the timestamps for a couple of published packages and the `integratedTime` was always slightly before the publish time in the package manfiest.

That said . . . when it comes time to do a key rotation I doubt that we're going to be able to identify the precise moment of key rotation (at least not down to the second/millisecond level of accuracy) so we're gonna want to give a wide margin of error to any `expires` value that we attach to any key when we rotate it. 

Knowing that we're gonna have a bit of a buffer in the expiry time, the difference between the `integratedTime` and the `mani._time` is probably unimportant. I'd say we should use whichever value is easiest to access.